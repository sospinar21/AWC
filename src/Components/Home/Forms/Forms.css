.forms {
  width: 30%;
  margin-left: 55%;
  padding: 10px;
  box-sizing: border-box;
  text-align: center;
  position: relative;
  height: 100vh; }
  .forms button, .forms input {
    display: block;
    border: .5px solid lightgrey;
    -webkit-box-flex: 1;
    -webkit-flex: 1 0 0;
    -ms-flex: 1 0 0;
    flex: 1 0 0px;
    margin-top: 20px;
    outline: 0;
    overflow: hidden;
    padding: 9px 0 7px 8px;
    text-overflow: ellipsis;
    background-color: #fafafa;
    width: 100%;
    box-sizing: border-box;
    height: 45px;
    font-size: 15px;
    cursor: pointer; }
  .forms h1 {
    color: #232b37;
    font-size: 50px;
    font-family: 'Kalam', cursive;
    text-shadow: 1px 1px 1px #100b0b; }
  .forms button {
    background-color: #b62d4d;
    color: white;
    height: 40px; }
  .forms button:disabled {
    background-color: #e6d9db;
    cursor: default; }
  .forms .selected {
    background-color: #232b37;
    box-shadow: 0px 2px 6px grey;
    border: 1px solid dimgrey; }
  .forms .no-selected {
    background-color: #232b37; }

.si-su {
  height: 100vh;
  overflow-y: hidden;
  background-color: #f3f3f3; }

.website {
  box-shadow: -1px 1px 13px lavenderblush;
  background-image: url("http://www.marinecitydancecompany.com//Cut%20out%20images/RED%20Jump%20cropped.png");
  background-size: cover;
  position: absolute;
  height: 900px;
  width: 45vw;
  margin: 0 240px 0;
  background-repeat: no-repeat; }

.forms-menu {
  display: flex; }
