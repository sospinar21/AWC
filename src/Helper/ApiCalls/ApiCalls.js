import React, {Component} from 'react';

class ApiCalls extends Component {
  constructor(){
    super()
  }

  fetchEvents = async () => {
    const url = `https://api.awc.dance/events?city=vermont`;
    const response = await fetch(url)
    const data= await response.json();
    return data;
  } 

  fetchStudios = () => {
    const url = `https://api.awc.dance/?city=80202`;
    return fetch(url, {
      method: 'GET',
      headers: {
        'Content-type': 'application/json',
      }
    }).then( response => response.json() )
      .then( data => data )   
  }

  fetchVideos= async () => {
    var url = 'https://api.awc.dance/youtubevideos'
    const response = await fetch(url)
    const data = await response.json()
    return data
  }
}

export default ApiCalls;