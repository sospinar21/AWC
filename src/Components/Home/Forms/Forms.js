import React, { Component } from 'react';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';
import './Forms.css';
import { NavBar } from '../NavBar/NavBar'
import  Login  from '../Login/Login';
import { SignUp } from '../SignUp/SignUp';

export class Forms extends Component {
  constructor() {
    super()
    this.state = {
      logIn: false,
      signUp: true
    }
  }

  showLogIn = () => {
    this.setState({ logIn: true, signUp: false})
  }

  showSignUp = () => {
    this.setState({ logIn: false, signUp: true})
  }

  displaySelected = () => {
    return this.state.logIn === true ? <Login /> : <SignUp />
  }

  render () { 
    const selected = this.state.logIn === true ? 'selected' : 'no-selected'
    const selected2 = this.state.signUp === true ? 'selected' : 'no-selected'

    return (
      <div className='si-su'>
        <NavBar />
        <div className='website'/> 
        <div className='forms'>
          <div className='forms-menu'> 
            <button className= {selected} onClick={this.showLogIn}>Log In </button>
            <button className={selected2} onClick={this.showSignUp}>Sign Up </button>
          </div>
          {this.displaySelected()}
        </div>
      </div>
    );
  }
}

export const mapStateToProps = (state) => {
  return ({
    suggestedEvents: state.suggestedEvents
  });
};

export const mapDispatchToProps = dispatch => ({
  
});

Forms.propTypes = {
  suggestedEvents: PropTypes.array,
};

export default connect(mapStateToProps, mapDispatchToProps)(Forms);