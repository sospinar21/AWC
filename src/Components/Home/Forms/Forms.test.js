import { Forms, mapStateToProps } from '../Forms/Forms'
import React from 'react';
import ReactDOM from 'react-dom';
import { mount, shallow } from 'enzyme';
import  ApiCalls  from '../../../Helper/ApiCalls/ApiCalls';


describe('forms', () => {
  let forms;
  let mockProps;

  beforeEach(() => {

    mockProps = {
      suggestedEvents: [{name: 'party'}],
    }

    forms = shallow(<Forms {...mockProps} /> )

  })

  it('matches snapshot', () => {
    expect(forms).toMatchSnapshot();
  })

  describe('showLogIn', () => {

    it('sets the the state to login: true', () => {

      forms.state().logIn = false

      forms.instance().showLogIn()

      expect(forms.state().logIn).toEqual(true)
    })
    
  })

  it('sets the the state to signUp: false', () => {

    forms.state().signUp = true

    forms.instance().showLogIn()

    expect(forms.state().signUp).toEqual(false)
  })

  describe('showSignUp', () => {

    it('sets the the state to login: false', () => {

      forms.state().logIn = true

      forms.instance().showSignUp()

      expect(forms.state().logIn).toEqual(false)
    })
    
  })

  it('sets the the state to signUp: true', () => {

    forms.state().signUp = false

    forms.instance().showSignUp()

    expect(forms.state().signUp).toEqual(true)
  })

  describe('mapStateToProps', () => {
    
    let suggestedEvents;
    let mappedProps

    it('returns an object with the suggestedEvents', () => {

      const mockState = {
        suggestedEvents: {name: 'Workshop'}
      }

      const mappedProps = mapStateToProps(mockState);
      expect(mappedProps).toEqual(mockState);
      
    })
  })
})
