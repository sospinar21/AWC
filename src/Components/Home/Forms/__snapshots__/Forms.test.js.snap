// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`forms matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Forms
    suggestedEvents={
        Array [
            Object {
              "name": "party",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="nav"
>
          <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    className="logo"
                    to="/"
          >
                    <div
                              className="logocir"
                    >
                              AWC
                    </div>
          </NavLink>
          <div
                    className="buffer"
          />
          <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    className="signink"
                    to="/"
          >
                    <div
                              className="txt"
                    >
                              Sign Up / SignIn
                    </div>
                    <div
                              className="ico"
                    >
                              <svg
                                        height="24"
                                        viewBox="0 0 24 24"
                                        width="24"
                                        xmlns="http://www.w3.org/2000/svg"
                              >
                                        <path
                                                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                                        />
                                        <path
                                                  d="M0 0h24v24H0z"
                                                  fill="none"
                                        />
                              </svg>
                    </div>
          </NavLink>
</div>,
        <div
          className="website"
/>,
        <div
          className="forms"
>
          <div
                    className="forms-menu"
          >
                    <button
                              className="no-selected"
                              onClick={[Function]}
                    >
                              Log In 
                    </button>
                    <button
                              className="selected"
                              onClick={[Function]}
                    >
                              Sign Up 
                    </button>
          </div>
          <SignUp />
</div>,
      ],
      "className": "si-su",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              className="logo"
              to="/"
>
              <div
                            className="logocir"
              >
                            AWC
              </div>
</NavLink>,
            <div
              className="buffer"
/>,
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              className="signink"
              to="/"
>
              <div
                            className="txt"
              >
                            Sign Up / SignIn
              </div>
              <div
                            className="ico"
              >
                            <svg
                                          height="24"
                                          viewBox="0 0 24 24"
                                          width="24"
                                          xmlns="http://www.w3.org/2000/svg"
                            >
                                          <path
                                                        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                                          />
                                          <path
                                                        d="M0 0h24v24H0z"
                                                        fill="none"
                                          />
                            </svg>
              </div>
</NavLink>,
          ],
          "className": "nav",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "children": <div
                className="logocir"
>
                AWC
</div>,
              "className": "logo",
              "to": "/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "AWC",
                "className": "logocir",
              },
              "ref": null,
              "rendered": "AWC",
              "type": "div",
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "buffer",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "children": Array [
                <div
                  className="txt"
>
                  Sign Up / SignIn
</div>,
                <div
                  className="ico"
>
                  <svg
                                    height="24"
                                    viewBox="0 0 24 24"
                                    width="24"
                                    xmlns="http://www.w3.org/2000/svg"
                  >
                                    <path
                                                      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                                    />
                                    <path
                                                      d="M0 0h24v24H0z"
                                                      fill="none"
                                    />
                  </svg>
</div>,
              ],
              "className": "signink",
              "to": "/",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sign Up / SignIn",
                  "className": "txt",
                },
                "ref": null,
                "rendered": "Sign Up / SignIn",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <svg
                    height="24"
                    viewBox="0 0 24 24"
                    width="24"
                    xmlns="http://www.w3.org/2000/svg"
>
                    <path
                                        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                    />
                    <path
                                        d="M0 0h24v24H0z"
                                        fill="none"
                    />
</svg>,
                  "className": "ico",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <path
                        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
/>,
                      <path
                        d="M0 0h24v24H0z"
                        fill="none"
/>,
                    ],
                    "height": "24",
                    "viewBox": "0 0 24 24",
                    "width": "24",
                    "xmlns": "http://www.w3.org/2000/svg",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "d": "M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "path",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "d": "M0 0h24v24H0z",
                        "fill": "none",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "path",
                    },
                  ],
                  "type": "svg",
                },
                "type": "div",
              },
            ],
            "type": [Function],
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "website",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="forms-menu"
>
              <button
                            className="no-selected"
                            onClick={[Function]}
              >
                            Log In 
              </button>
              <button
                            className="selected"
                            onClick={[Function]}
              >
                            Sign Up 
              </button>
</div>,
            <SignUp />,
          ],
          "className": "forms",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <button
                  className="no-selected"
                  onClick={[Function]}
>
                  Log In 
</button>,
                <button
                  className="selected"
                  onClick={[Function]}
>
                  Sign Up 
</button>,
              ],
              "className": "forms-menu",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Log In ",
                  "className": "no-selected",
                  "onClick": [Function],
                },
                "ref": null,
                "rendered": "Log In ",
                "type": "button",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sign Up ",
                  "className": "selected",
                  "onClick": [Function],
                },
                "ref": null,
                "rendered": "Sign Up ",
                "type": "button",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="nav"
>
            <NavLink
                        activeClassName="active"
                        ariaCurrent="true"
                        className="logo"
                        to="/"
            >
                        <div
                                    className="logocir"
                        >
                                    AWC
                        </div>
            </NavLink>
            <div
                        className="buffer"
            />
            <NavLink
                        activeClassName="active"
                        ariaCurrent="true"
                        className="signink"
                        to="/"
            >
                        <div
                                    className="txt"
                        >
                                    Sign Up / SignIn
                        </div>
                        <div
                                    className="ico"
                        >
                                    <svg
                                                height="24"
                                                viewBox="0 0 24 24"
                                                width="24"
                                                xmlns="http://www.w3.org/2000/svg"
                                    >
                                                <path
                                                            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                                                />
                                                <path
                                                            d="M0 0h24v24H0z"
                                                            fill="none"
                                                />
                                    </svg>
                        </div>
            </NavLink>
</div>,
          <div
            className="website"
/>,
          <div
            className="forms"
>
            <div
                        className="forms-menu"
            >
                        <button
                                    className="no-selected"
                                    onClick={[Function]}
                        >
                                    Log In 
                        </button>
                        <button
                                    className="selected"
                                    onClick={[Function]}
                        >
                                    Sign Up 
                        </button>
            </div>
            <SignUp />
</div>,
        ],
        "className": "si-su",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <NavLink
                activeClassName="active"
                ariaCurrent="true"
                className="logo"
                to="/"
>
                <div
                                className="logocir"
                >
                                AWC
                </div>
</NavLink>,
              <div
                className="buffer"
/>,
              <NavLink
                activeClassName="active"
                ariaCurrent="true"
                className="signink"
                to="/"
>
                <div
                                className="txt"
                >
                                Sign Up / SignIn
                </div>
                <div
                                className="ico"
                >
                                <svg
                                                height="24"
                                                viewBox="0 0 24 24"
                                                width="24"
                                                xmlns="http://www.w3.org/2000/svg"
                                >
                                                <path
                                                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                                                />
                                                <path
                                                                d="M0 0h24v24H0z"
                                                                fill="none"
                                                />
                                </svg>
                </div>
</NavLink>,
            ],
            "className": "nav",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": <div
                  className="logocir"
>
                  AWC
</div>,
                "className": "logo",
                "to": "/",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "AWC",
                  "className": "logocir",
                },
                "ref": null,
                "rendered": "AWC",
                "type": "div",
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "buffer",
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": Array [
                  <div
                    className="txt"
>
                    Sign Up / SignIn
</div>,
                  <div
                    className="ico"
>
                    <svg
                                        height="24"
                                        viewBox="0 0 24 24"
                                        width="24"
                                        xmlns="http://www.w3.org/2000/svg"
                    >
                                        <path
                                                            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                                        />
                                        <path
                                                            d="M0 0h24v24H0z"
                                                            fill="none"
                                        />
                    </svg>
</div>,
                ],
                "className": "signink",
                "to": "/",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Sign Up / SignIn",
                    "className": "txt",
                  },
                  "ref": null,
                  "rendered": "Sign Up / SignIn",
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <svg
                      height="24"
                      viewBox="0 0 24 24"
                      width="24"
                      xmlns="http://www.w3.org/2000/svg"
>
                      <path
                                            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                      />
                      <path
                                            d="M0 0h24v24H0z"
                                            fill="none"
                      />
</svg>,
                    "className": "ico",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <path
                          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
/>,
                        <path
                          d="M0 0h24v24H0z"
                          fill="none"
/>,
                      ],
                      "height": "24",
                      "viewBox": "0 0 24 24",
                      "width": "24",
                      "xmlns": "http://www.w3.org/2000/svg",
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "d": "M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z",
                        },
                        "ref": null,
                        "rendered": null,
                        "type": "path",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "d": "M0 0h24v24H0z",
                          "fill": "none",
                        },
                        "ref": null,
                        "rendered": null,
                        "type": "path",
                      },
                    ],
                    "type": "svg",
                  },
                  "type": "div",
                },
              ],
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "website",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="forms-menu"
>
                <button
                                className="no-selected"
                                onClick={[Function]}
                >
                                Log In 
                </button>
                <button
                                className="selected"
                                onClick={[Function]}
                >
                                Sign Up 
                </button>
</div>,
              <SignUp />,
            ],
            "className": "forms",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <button
                    className="no-selected"
                    onClick={[Function]}
>
                    Log In 
</button>,
                  <button
                    className="selected"
                    onClick={[Function]}
>
                    Sign Up 
</button>,
                ],
                "className": "forms-menu",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Log In ",
                    "className": "no-selected",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "Log In ",
                  "type": "button",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Sign Up ",
                    "className": "selected",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "Sign Up ",
                  "type": "button",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
